### PR TITLE
Add `size-*` shorthand

### PR DESCRIPTION
**Pull request by thecrypticace**
This PR adds a `size-*` utility that sets both `width` and `height`.

The supported values are the same as the `w-*` and `h-*` utilities with the exception of screen units (`screen`, `svh`, `lvh`, `dvh`, `svw`, `lvw`, and `dvw`).

Now you can write this:
```html
<div class="size-12"></div>
```

And it will generate the following css:
```css
.size-12 {
  width: 3rem;
  height: 3rem;
}
```

### CODE REVIEW

File: tests/plugins/size.test.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { quickPluginTest } from '../util/run'
+
+quickPluginTest('size', {
+  safelist: [
+    // Arbitrary values
+    's-[12px]',
+    's-[var(--my-value)]',
```
[the remark this answers is not in the document]
Comment (thecrypticace) [reply tests/plugins/size.test.js:7]: heh yup. Fixed.

File: stubs/config.full.js [hunk 2]
```diff
@@ -968,6 +968,40 @@ module.exports = {
       '3/4': '75%',
       full: '100%',
     }),
+    size: ({ theme }) => ({
```
Comment (ArnaudBarre) [inline stubs/config.full.js:971]: NIT: seems like the alphabetical order was not respected

**Comment by yassernasc** [comment]
Cool! But why not a shorthand for `size-screen`?

**Comment by A7med3bdulBaset** [comment]
> Cool! But why not a shorthand for `size-screen`?

Because, there is no `h-screen`

**Comment by DamianGlowala** [comment]
@thecrypticace why were the aforementioned exceptions excluded?

**Pull request merged**